<launch>
	
	<!-- Run the MAP SERVER -->
	<node name="laserScannerMatcher" pkg="laser_scan_matcher" type="laser_scan_matcher_node" output="screen">
		
		<!-- The frame to set in the header of the published map.  -->
		<param name="fixed_frame" value="map" />
		<param name="base_frame" value="base_footprint" />
		<param name="use_imu" value="false" />
		<param name="use_odom" value="false" />
		<param name="publish_pose" value="false"/>
		<param name="publish_pose_stamped" value="true"/>
		<param name="kf_dist_linear" value="0.05"/>
		<param name="kf_dist_angular" value="0.175"/>
		<param name="publish_tf" value="false"/>

	</node>
		

</launch>